setup.py

```diff
@@ -31,7 +31,9 @@ def get_file_contents(file_path):
       install_requires=["requests",
                         "pysqlite",
                         #"usb",
-                        "qrcode"],
+                        "qrcode",
+                        "cffi",
+                        "enum"],
```

Comment:
enum or enum34? I would recommend enum34, which is available as debian package on ubuntu.